src/main/resources/data/nucleoid_extras/lang/fr_fr.json

```diff
@@ -0,0 +1,7 @@
+{	
+	"nucleoid.sidebar.welcome": "Bienvenue sur Nucleoid !",
+	"nucleoid.sidebar.games": "Jeux ouverts:",
+	"nucleoid.sidebar.join": "...exécutez /game join",
+	"nucleoid.sidebar.compass": "ou utilisez la boussole !",
+	"nucleoid.discord": "nucleoid.xyz/discord"
```

Comment:
```suggestion
	"nucleoid.discord": "nucleoid.xyz/discord",
	"nucleoid.stop.scheduled": "Le redémarrage du serveur a été programmé ! Le serveur va redémarrer dans %s minutes, ou dès que toutes les parties seront terminées.",
        "nucleoid.stop.scheduled.already": "Un redémarrage du serveur a déja été programmé.",
        "nucleoid.stop.game.open": "Impossible de rejoindre cette partie: un redémarrage a été programmé !"
```

Reply:
:thinking: I would rather say `Un redémarrage` in both cases but I need Lambda opinion on this.
We (Lambda and I) have already used `jeux` to translate `games` in plasmid, so I think we should do the same here.
Also you are missing a `à` in `déja`.

Reply:
I didn't make other translations, so you'll be better than me for that, but there can't be multiple reboot, so Le or Un could potentially be used interchangeably